drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -143,11 +146,21 @@ static int _recv(netdev_t *netdev, void *buf, size_t len, void *info)
     /* ignore MSB (refer p.80) and substract length of FCS field */
     pkt_len = (phr & 0x7f) - 2;
 
-    /* just return length when buf == NULL */
+    /* return length when buf == NULL */
     if (buf == NULL) {
+        /* release SPI bus */
         at86rf2xx_fb_stop(dev);
+
+        /* drop packet, continue receiving */
+        if (len > 0) {
+            /* set device back in operation state which was used before last transmission.
+             * e.g RX_AACK_ON */
+            at86rf2xx_set_state(dev, dev->idle_state);
```

Comment:
I'm not really sure about this state, who sets it (e.g. `RX_AACK_ON`)?

Reply:
https://github.com/RIOT-OS/RIOT/blob/4c753ece542951d0ba4313a3f914972ffe4f228b/drivers/at86rf2xx/at86rf2xx.c#L112

It is set her and when starting to transmit, then the old state is saved.